azurerm/resource_arm_cosmosdb_mongo_collection.go

```diff
@@ -118,20 +120,27 @@ func resourceArmCosmosDbMongoCollectionCreateUpdate(d *schema.ResourceData, meta
 	account := d.Get("account_name").(string)
 	database := d.Get("database_name").(string)
 	throughput := d.Get("throughput").(int)
+	dbHasThroughputConfigured := throughput > 0
```

Comment:
If we remove the default we can do:
```suggestion
	throughput , hasThroughput := d.GetOk("throughput)
```